src/app/diagnostics/logger.ts

```diff
@@ -44,16 +45,34 @@ class ConsoleLogger implements Logger {
     }
 }
 
+export function instrument<T>(logger: Logger, name: string): OperatorFunction<T, T> {
```

Comment:
please add some comment how we can use this utility function in debugging time and we don't use actually at this point in time.